modules/openapi-generator/src/main/resources/elixir/mix.exs.mustache

```diff
@@ -34,4 +36,12 @@ defmodule {{moduleName}}.Mixfile do
 {{/deps}}
     ]
   end
+
+   defp package() do
+    [
+      name: "{{#underscored}}{{packageName}}{{/underscored}}",
+      files: ~w(lib mix.exs README* LICENSE*),
+      licenses: ["{{licenseId}}"]
```

Comment:
@tobbbles I couldn't find a way to set `licenseId ` - do you know how we can set that value?

Reply:
That was easy enough ... https://github.com/ory/sdk/commit/20f602a6a52bd3dbb3e2db3ba1eef574d7cc45fb